pkg/js/helpers.js

```diff
@@ -36,7 +36,27 @@ function getConfiguredDomains() {
     return conf.domain_names;
 }
 
-function NewRegistrar(name, type, meta) {
+// NewRegistrar returns an registrar object.
+// For backwards compatibility, it accepts (name), (name, meta),
+// (name, type), (name, type, meta).
+function NewRegistrar() {
+  // For backwards compatibility, this is a wrapper around the legacy
+  // version of this function.
+  switch (arguments.length) {
+    case 1:
+      return oldNewRegistrar(arguments[0], "-")
+      break;
```

Comment:
Is `break` after `return` a stylistic choice?  (My JS is poor so I'm a little confused)

Reply:
Nope.  Just a typo.